x-pack/plugins/telemetry_collection_xpack/README.md

```diff
@@ -0,0 +1,9 @@
+# telemetry_collection_xpack
+
+> Add the telemetry collector with the xpack information
```

Comment:
?

Reply:
Changed the description into something that hopefully explains it better.